tests/App/Document/Address.php

```diff
@@ -16,16 +16,14 @@
 use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM;
 
 /** @ODM\EmbeddedDocument() */
```

Comment:
Does attributes exists for ODM ?
Since we dropped 7.4, we could use attribute everywhere.

Reply:
yep, I was doing that